extension/src/popup/components/manageAssets/ChooseAsset/index.tsx

```diff
@@ -89,6 +92,24 @@ export const ChooseAsset = ({ balances }: ChooseAssetProps) => {
             image: "",
             domain: "",
           });
+
+          if (isExperimentalModeEnabled && sorobanBalances.length) {
+            sorobanBalances.forEach(({ symbol, contractId, name }) => {
+              // TODO:
```

Comment:
Interesting interaction, made me think a bit more about how we might want to handle values coming from a smart contract that could now break the interface but haven't found the right way to escape this yet.